src/types/registration.types.ts

```diff
@@ -277,6 +277,8 @@ export interface RegistrationAggregations {
   contributor?: AggregationValue[];
   publisher?: AggregationValue[];
   series?: AggregationValue[];
+  journal?: AggregationValue[];
+  hasPublicFile?: AggregationValue[];
```

Comment:
Jeg får `hasFile` i responsen fra API ser jeg, ikke `hasPublicFile` 🤔 Noe som jobbes med?

Reply:
https://github.com/BIBSYSDEV/nva-search-api/commit/32b5243232b93265c70fbb878493a267df2607e2

Sjekket inn for 20 min siden :)

Reply:
Den innsjekken fikser forhpentligvis kommentaren min over også

Reply:
Ser ut som det er implementert som to ulike fasetter der da: `hasPublicFile` og `noPublicFile` 🤔

Reply:
Ah, de har gjort det også ja. Skal oppdatere 👍